src/game/hud.c

```diff
@@ -446,7 +446,8 @@ LangArray textTime = DEFINE_LANGUAGE_ARRAY(
     "TIME %0d'%02d\"%d",
     "TEMPS %0d'%02d\"%d",
     "ZEIT %0d'%02d\"%d",
-    "TIME %0d'%02d\"%d");
+    "TIME %0d'%02d\"%d",
+    "TIEM.%0d'%02d\"%d");
 
```

Comment:
French has five letters and a space, should this be `TIEM. ` (space after period)

Reply:
that makes it move into the HP meter slightly (but the same thing happens with french)

idk what do you prefer?

![imagen](https://user-images.githubusercontent.com/15913880/207471703-390385ba-5378-476d-97d2-b3d88ca0743c.png)

![imagen](https://user-images.githubusercontent.com/15913880/207471804-48078b06-99f9-408c-926c-ed88b91d474a.png)

Reply:
i guess it's fine if french does it, will change it

Reply:
done

Reply:
so you're saying anything is fine if french does it 👀

Reply:
death